docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,443 @@
+   - `<RELATIVE-PATH-TO-TLS-CERT>` - with the path to the tls-cert.pem file that you copied from your TLS CA. This is the path relative to `FABRIC_CA_CLIENT_HOME`.
+   - `<CA_HOSTNAME>` - with a comma-separated list of host names for which the certificate should be valid. If not specified, the default value from the `fabric-ca-client-config.yaml` is used. If a host name is dynamic you can specify a wildcard for the domain. For example, when you include the flag `--csr.hosts 'host1,*.example.com'` it means that the hostname `host1` is recognized as well as any host from the `example.com` domain.
+
+   In this case, the `-d` parameter runs the client in DEBUG mode which is useful for debugging command failures.
+
+   For example:
+   ```
+   fabric-ca-client enroll -d -u https://tls-admin:tls-adminpw@my-machine.example.com:7054 --tls.certfiles ../crypto/ca-cert.pem --enrollment.profile tls --csr.hosts 'host1,*.example.com'
+   ```
+
+   When this command completes, an `/msp` folder is generated under `/tls-ca/admin` and contains the TLS signing cert and private key for the node.
+
+   **Important:** Make note of the `/msp/signcerts/cert.pem` and `/msp/keystore/` files. These are your TLS signcert and private key that you will need to reference in the CA configuration .yaml file in the following steps.
+
+   The `--enrollment.profile tls` flag is specified because we are enrolling against the TLS CA. Use of this flag means that the enrollment is performed according to the `usage` and `exipry` settings of the TLS profile that is defined in the `signing` section of the configuration .yaml file.
+
+
+The following diagram is a conceptual summary of the steps you performed to create a TLS CA server:
+
+![ca-tls-flow](./ca-tls-flow.png)
+
+### (Optional) Register the Intermediate CA admin with the TLS CA
+
+If you are planning to have an intermediate CA in addition to a root CA, you can also register and enroll that intermediate CA admin user now as well. The following command registers the intermediate ca admin id `icaadmin` and `icaadminpw` with the TLS CA.
+
+```
+fabric-ca-client register -d --id.name icaadmin --id.secret icaadminpw -u https://my-machine.example.com:7054  --tls.certfiles ../crypto/ca-cert.pem --csr.hosts 'host1,*.example.com'
+```
+
+Now would also be a good time to generate the intermediate CA TLS certificates. The `-M` flag on the enroll command is used to designate where the generated intermediate CA TLS certificates should be stored for the `icaadmin` user. Because these certificates are for a new identity, it is a best practice to put them in a different folder than the `admin` identity. Therefore, instead of generating them in the default `msp` folder, we will put them in a new folder named `msp-ica`.
+
+```
+fabric-ca-client enroll -d -u https://icaadmin:icaadminpw@my-machine.example.com:7054 --tls.certfiles ../crypto/ca-cert.pem -M msp-ica --csr.hosts 'host1,*.example.com'
+```
```

Comment:
The way you do i now, `-M msp-ica`, puts the tls certs used by the intermediate ca in the admin folder of of the ca admin user, which i found a bit cumbersome. One of the benefits of using the full path is that you can set  `FABRIC_CA_CLIENT_HOME` once, and then put everything else where it makes sense.

Reply:
I've taken your advice and refactored the folder locations and now simply set the FABRIC_CA_CLIENT_HOME to be the fully qualified path to the location of the fabric ca client binary file.